libs/shared/src/types/feature-flags/feature-flags.ts

```diff
@@ -14,4 +14,5 @@ export enum FeatureFlagsKeysEnum {
   IS_V2_EXPERIENCE_ENABLED = 'IS_V2_EXPERIENCE_ENABLED',
   IS_MIXPANEL_RECORDING_ENABLED = 'IS_MIXPANEL_RECORDING_ENABLED',
   IS_INTEGRATION_INVALIDATION_DISABLED = 'IS_INTEGRATION_INVALIDATION_DISABLED',
+  IS_V2_ENABLED = 'IS_V2_ENABLED',
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** Should we remove `IS_V2_EXPERIENCE_ENABLED` then?‏